src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="System.IO.Ports" Version="$(SystemIOPortsPackageVersion)" />
     <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj" />
     <PackageReference Include="UnitsNet" Version="$(UnitsNetPackageVersion)" />
+    <PackageReference Include="System.Management" Version="5.0.0-preview.5.20224.13" />
```

Comment:
See my other comment, basically versions should be comming from Versions.props instead of this project defining them. If you want me to do that let me know. Also, to answer @krwq question: there is no such thing as a platform specific dependency on a NuGet package.